src/kubernetes.cc

```diff
@@ -1137,13 +1157,24 @@ bool KubernetesUpdater::ValidateConfiguration() const {
 
 void KubernetesUpdater::StartUpdater() {
   if (config().KubernetesUseWatch()) {
-    // Wrap the bind expression into a function to use as a bind argument.
-    UpdateCallback cb = std::bind(&KubernetesUpdater::MetadataCallback, this,
-                                  std::placeholders::_1);
-    node_watch_thread_ =
-        std::thread(&KubernetesReader::WatchNode, &reader_, cb);
-    pod_watch_thread_ =
-        std::thread(&KubernetesReader::WatchPods, &reader_, cb);
+    const std::string current_node = reader_.CurrentNode();
+
+    if (config().VerboseLogging()) {
+      LOG(INFO) << "Current node is " << current_node;
+    }
+
+    const std::string watched_node(
```

Comment:
If we don't have it already, can we add a very prominent log line (not based on verbose) that states that we're doing a cluster level watch, vs a node level watch?

Reply:
Done.